File: Javascript/Algorithms/9.js
let isPalindrome = function(x) {
  if (x < 0) return false
    let s = `${x}`, [i, j] = [0, s.length - 1];
  while (i < j) {
    if (s[i++] != s[j--]) return false
  }
  return true;
};

File: Python/Algorithms/9.py
class Solution:
    def isPalindrome(self, n):
        if n < 0:
            return False
        s = str(n)
        length = len(s)
        i, j = 0, length-1
        while i < j:
            if s[i] != s[j]:
                return False
            i += 1
            j -= 1
        return True
